### PR TITLE
Knex processor

### PR DESCRIPTION
**Pull request by Exelord**

### CODE REVIEW

File: src/knex-processor.ts [hunk 1]
```diff
@@ -0,0 +1,54 @@
+import * as Knex from "knex";
+
+import OperationProcessor from "./operation-processor";
+import { Resource } from "./types";
+
+export default class KnexProcessor<
+  ResourceT extends Resource
+> extends OperationProcessor<ResourceT> {
+  private knex: any;
```
Comment (joelalejandro) [inline src/knex-processor.ts]: @Exelord can we be explicit about the type for this property? It should be:

```suggestion
  private knex: Knex;
```

File: src/knex-processor.ts [hunk 2]
```diff
@@ -0,0 +1,54 @@
+import * as Knex from "knex";
```
Comment (joelalejandro) [inline src/knex-processor.ts]: @Exelord A regular `import` (no wildcard) should work:

```suggestion
import Knex from "knex";
```

If it doesn't, can you verify that `allowSyntheticImports` is set to `true` on the `tsconfig.json` file?

**Comment by joelalejandro** [comment]
Seems that this PR is no longer necessary due to #10?

File: src/knex-processor.ts (hunk 2 above)
Comment (Exelord) [reply src/knex-processor.ts]: It is present by the `*` is still required to have it working :/

**Comment by Exelord** [comment]
Yes, you right!

**Pull request closed without merging**